src/api/status/public/js/build-log/build-header.js

```diff
@@ -0,0 +1,39 @@
+const buildHeaderTitle = document.getElementById('build-header-title');
+const buildHeaderInfo = document.getElementById('build-header-info');
+const buildSender = document.getElementById('build-sender');
+const buildSenderName = document.getElementById('build-sender-name');
+const buildSenderImg = document.getElementById('build-sender-img');
+const buildGitSHA = document.getElementById('build-git-sha');
+const buildResult = document.getElementById('build-result');
+const buildStarted = document.getElementById('build-started');
+const buildDuration = document.getElementById('build-duration');
+
+function renderBuildInfo({ githubData, startedAt, stoppedAt, result }) {
+  buildHeaderTitle.innerHTML = '';
+  buildHeaderTitle.innerText = 'Build Status';
```

Comment:
overwriting `innerHtml` should be enough
```suggestion
  buildHeaderTitle.innerHTML = 'Build Status';
```